src/containers/shared/components/Transaction/EscrowFinish/test/mock_data/EscrowFinish.json

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
You should delete `src/containers/Transactions/test/mock_data/EscrowFinish.json`